### PR TITLE
docs: Fix broken links in the documents

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: lakshmipathi.ganapathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

File: kernelci.org/content/en/docs/instances/staging.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ There is a timer on the staging.kernelci.org server which starts a job every
 1. update the `kernelci-frontend` service using Ansible from [`kernelci-frontend-config`](https://github.com/kernelci/kernelci-frontend-config) with the staging branch
 1. create and push a `staging.kernelci.org` branch with a tag to the [KernelCI
    kernel repo](https://github.com/kernelci/linux)
-1. trigger a monitor job in Jenkins with the [`kernelci_staging`](https://github.com/kernelci/kernelci-core/blob/staging.kernelci.org/build-configs.yaml#L612) config
+1. trigger a monitor job in Jenkins with the [`kernelci_staging`](INSERT CORRECT LINK HERE) config
```
Comment (Lakshmipathi) [inline kernelci.org/content/en/docs/instances/staging.md:85]: @gctucker what should be URL here?

Comment (gctucker) [reply kernelci.org/content/en/docs/instances/staging.md:85]: Same path as production but on the `staging.kernelci.org` branch

Comment (gctucker) [reply kernelci.org/content/en/docs/instances/staging.md:85]: @Lakshmipathi Please fix this link so we can merge the PR.
```suggestion
1. trigger a monitor job in Jenkins with the [`kernelci_staging`](https://github.com/kernelci/kernelci-core/blob/staging.kernelci.org/config/core/build-configs.yaml#L836) config
```
In fact there are now 3 kernel branches created by the KernelCI staging jobs based on mainline, stable and next.  But we can merge this PR and then update the page accordingly as a follow-up.

Comment (Lakshmipathi) [reply kernelci.org/content/en/docs/instances/staging.md]: Applied this change.

**Review by gctucker — APPROVED** [review]
Verified OK on staging.

**Pull request merged**